### PR TITLE
Begin wireframing components

### PR DESCRIPTION
**Pull request by ccali11**
Creating a PR to keep it bite sized. This branch sets up and renders parent component via express static method. Going to keep working on this branch because if I didn't, I'd be blocked.

@CSellers720 - If you have a moment, please take a glance, otherwise, feel free to wait until tomorrow to talk about process here because it is a little unclear if I should have not made a PR and kept working (at the expense of creating a much larger PR chunk). It's simple code and just wireframing, so not looking for a detailed review.

### CODE REVIEW

**Comment by CSellers720** [comment]
I was attempting to make sub-branches so that each PR was relatively small and I could keep building off of the same code. It was working pretty well until I tried to merge the tip and it folded everything in together. You can use the command `git checkout -b childBranchName parentBranchName` to create sub-branches.

File: client/App.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import $ from 'jquery';
+
+import Stats from './components/Stats.jsx';
+import Sort from './components/Sort.jsx';
+import ReviewsFeed from './components/ReviewsFeed.jsx';
+
+class App extends React.Component {
```
Comment (CSellers720) [inline client/App.jsx:9]: Have you looked into Hooks? They're just as functional as class components and make states a lot easier/faster to build and pass between sub-components. Not required by any means, but I'd encourage you to look into it! They're really quick to pick up.

File: client/components/Review.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const SingleReview = styled.div`
+  display: flex;
+  border: 1px solid black;
+  flex-direction: column;
+`;
```
Comment (CSellers720) [inline client/components/Review.jsx:8]: I see you went with styled components as well! How are you liking them? I was considering moving some of my styling blocks to a separate `.js` file and importing them into my components to make things more readable.

File: package.json [hunk 3]
```diff
@@ -4,9 +4,9 @@
   "description": "Creating a clone of an item page of the brilliant earth engagement ring website with a team of four. This repo is for the customer reviews component.",
   "private": true,
   "scripts": {
-    "build": "webpack",
+    "build": "webpack -d --watch --config webpack.config.js",
```
Comment (CSellers720) [inline package.json:7]: Looks good. Checking `webpack.config.js` next.

**Comment by ccali11** [comment]
image

^Screenshot to demarcate where I am in my wireframe rendering

File: webpack.config.js [hunk 4]
```diff
@@ -17,6 +17,14 @@ module.exports = {
             presets: ['@babel/preset-env', '@babel/preset-react']
           }
         }
+      },
+      {
+        test: /\.(png|jpe?g|gif)$/i,
+        use: [
+          {
+            loader: 'file-loader',
+          },
```
Comment (CSellers720) [inline webpack.config.js:26]: What files are you intending to load? I just stuck everything in my S3 bucket and use URLs so I don't have to serve anything up from my server aside from the html and js files, though this isn't the only way to do it.

File: webpack.config.js [hunk 5]
```diff
@@ -17,6 +17,14 @@ module.exports = {
...
+        ],
```
Comment (CSellers720) [inline webpack.config.js:27]: Looks like you might be missing a `}` here.

**Review by CSellers720 — APPROVED** [review]
Looks like everything is coming along. Specific comments and questions are in my review.

File: client/App.jsx (hunk 1 above)
Comment (ccali11) [reply client/App.jsx:9]: @CSellers720 - Thank you for this comment! I don't have any experience with React Hooks yet, so figured I'd start without them. August's presentation inspired me to learn Hooks though, so was planning on refactoring later this week if there was time. I'll give them a glance today before I go any further and make a call of whether I want to use them or not.

File: client/components/Review.jsx (hunk 2 above)
Comment (ccali11) [reply client/components/Review.jsx:8]: @CSellers720 - Seemed more intuitive than CSS Modules to me, although I was reading that other people disagree vehemently haha. That definitely sounds like a good readability technique! I will keep in mind as well.

File: webpack.config.js (hunk 4 above)
Comment (ccali11) [reply webpack.config.js:26]: @CSellers720 - This is a good example of when I almost don't want a code review; I was just testing out a feature in my repo. If I was more savvy with git, I would have created that sub-branch, experimented, and then discarded/merged accordingly. Alas, I can only consume so much from the fire hose at once ;)

File: webpack.config.js (hunk 5 above)
Comment (ccali11) [reply webpack.config.js:27]: I didn't think so, but I'll take a look.

**Pull request merged**